data/jobs.txt

```diff
@@ -1573,3 +1573,192 @@ mission "Large Rush Delivery [3]"
+	on complete
+		dialog "You bid goodbye to the scientists and accept your payment of <payment>."
+		payment 100000 200
+
+mission "Escort science vessel"
+	name "Escort science vessel"
+	description "The crew of the science vessel <npc> requests an escort to <stopovers> where they plan to study some unusual phenomena. Escort them there and then return them to <planet> for <payment>."
+	job
+	repeat
+	to offer
+		random < 40
+		"combat rating" > 10
+	passengers 7 12
+	source
+		attributes research
+	stopover
+		attributes quarg pirate volcanic frontier "dirt belt" north south
+		distance 6 50
+	npc
+		system
+			distance 1 2
+		government Pirate
+		personality nemesis waiting plunders
+		fleet
+			names "pirate"
+			cargo 1
+			variant
+				"Sparrow"
+				"Sparrow"
+				"Sparrow"
+			variant
+				"Fury"
+				"Fury"
+			variant
+				"Hawk"
+				"Hawk"
+			variant
+				"Berserker"
+				"Fury"
+			variant
+				"Headhunter (Particle)"
+				"Sparrow"
+			variant
+				"Raven"
+			variant
+				"Corvette"
+	npc accompany save
+		government Merchant
+		personality escort timid
+		fleet
+			names "civilian"
+			variant
+				"Scout"
+			variant
+				"Bounder"
+	on stopover
+		dialog "The <npc> lands alongside you and teams of scientists wearing crisp white lab coats and serious-looking boots and gloves descend on the test site with all manner of exotic equipment. As they take their measurements, you prepare for the return journey to <planet>."
+	on visit
+		dialog "You have reached <planet>, but you left the <npc> behind! Better depart and wait for them to arrive in this star system."
+	on complete
+		dialog "You bid goodbye to the captain of the <npc> and accept your payment of <payment>."
+		payment 100000
+
+mission "Corporate espionage"
+	name "Corporate espionage"
+	description "The <npc> is about to enter this system. Locate it and make detailed scans of its cargo, which are of considerable interest to a team of Syndicate researchers on <destination>; proceed there for <payment> after retrieving the data."
+	job
+	repeat
+	to offer
+		random < 10
+		"reputation: Syndicate" > 100
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral"
+	destination
+		government "Syndicate"
+		attributes research
+		distance 7
+	npc "scan cargo"
+		government Merchant
+		personality timid fleeing uninterested entering
+		fleet
+			names "civilian"
+			variant
+				"Scout"
+			variant
+				"Bounder"
+		dialog "You manage to make the requested scans of the <npc>'s cargo. Time to head to <planet>."
+	on complete
+		dialog `You transmit your scans and receive a hearty congratulations, as well as <payment>. "This will set them back months!" You hear in the background.`
+		payment 45000
+
+mission "VIP transport"
```

Comment:
relates to https://github.com/endless-sky/endless-sky/pull/553 , i'd imagine VIP would expect a luxury transport only.